default.cfg.example

```diff
@@ -47,6 +47,9 @@ minloansize = 0.001
 #If this happens, KeepStuckOrders will keep your order where it is so maybe it can be filled. Otherwise it will be canceled and held until orders expire.
 keepstuckorders = True
 
+#Hide coins - Instead of keeping your coins lent out at minlendrate when it is not met, the bot will hold them and wait for the rate to surpass it.
+hideCoins = True
```

Comment:
At some point we will rename all the configuration settings, since they are a 50/50 split between lowercase or camelCase. 

Since it will be a breaking change we will need to enforce conventions after that point, but right now there is no point since there is no consistency.